pulpcore/tasking/tasks.py

```diff
@@ -113,35 +113,35 @@ def _queue_reserved_task(func, inner_task_id, resources, inner_args, inner_kwarg
 
     try:
         q = Queue(worker.name, connection=redis_conn)
-        q.enqueue(func, args=inner_args, kwargs=inner_kwargs, job_id=inner_task_id,
+        q.enqueue(func, args=inner_args, kwargs=inner_kwargs, job_id=inner_job_id,
                   timeout=TASK_TIMEOUT, **options)
     finally:
-        q.enqueue(_release_resources, args=(inner_task_id, ))
+        q.enqueue(_release_resources, args=(inner_job_id, ))
 
 
-def _release_resources(task_id):
+def _release_resources(job_id):
     """
     Do not queue this task yourself. It will be used automatically when your task is dispatched by
     the _queue_reserved_task task.
 
     When a resource-reserving task is complete, this method releases the task's resource(s)
 
     Args:
-        task_id (basestring): The UUID of the task that requested the reservation
+        job_id (basestring): The job_id of the task that requested the reservation
 
     """
     try:
-        task = Task.objects.get(pk=task_id, state=TASK_STATES.RUNNING)
+        task = Task.objects.get(job_id=job_id, state=TASK_STATES.RUNNING)
     except Task.DoesNotExist:
         pass
     else:
         msg = _('The task {task_id} exited immediately for some reason. Marking as '
                 'failed. Check the logs for more details')
-        _logger.error(msg.format(task_id=task_id))
-        exc = RuntimeError(msg.format(task_id=task_id))
+        _logger.error(msg.format(task_id=task.id))
+        exc = RuntimeError(msg.format(task_id=task.id))
```

Comment:
Same issue with 'id' as above.

Reply:
This one is ours, so this could become `pk` possible. I don't have a preference.